src/Pagerfanta/Pagerfanta.php

```diff
@@ -105,6 +105,11 @@ public function getNormalizeOutOfRangePages()
         return $this->normalizeOutOfRangePages;
     }
 
+    /**
```

Comment:
Documenting is also helpful for me to indicate the IDE things that can't be guessed.